data-view.js

```diff
@@ -0,0 +1,30 @@
+
+
+async function fetchData() {
+    try {
+        const response = await fetch('https://jsonplaceholder.typicode.com/todos/');
+        if (!response.ok) {
+            throw new Error('Network response was not ok');
+        }
+        const data = await response.json();
+        displayData(data);
```

Comment:
Best practice is to keep the Function above this function you are going to use.

Reply:
resolve